nb-extension/python/declarativewidgets/extensionapp.py

```diff
@@ -177,35 +187,90 @@ def start(self):
 
         self.log.info("Deactivating jupyter_declarativewidgets JS notebook extensions")
         self.section = "notebook"
-        self.disable_nbextension("urth_widgets/js/main")
+        self.disable_nbextension("declarativewidgets/js/main")
 
         self.log.info("Done. You may need to restart the Jupyter notebook server for changes to take effect.")
 
 
+class ExtensionQuickSetupApp(BaseNBExtensionApp):
+    """Installs and enables all parts of this extension"""
+    name = "jupyter declarativewidgets quick-setup"
+    version = __version__
+    description = "Installs and enables all features of the jupyter_declarativewidgets extension"
+
+    def start(self):
+        self.argv.extend(['--py', 'declarativewidgets'])
+
+        from notebook import serverextensions
+        install = serverextensions.EnableServerExtensionApp()
+        install.initialize(self.argv)
+        install.start()
+        from notebook import nbextensions
+        install = nbextensions.InstallNBExtensionApp()
+        install.initialize(self.argv)
+        install.start()
+        enable = nbextensions.EnableNBExtensionApp()
+        enable.initialize(self.argv)
+        enable.start()
+
+class ExtensionQuickRemovalApp(BaseNBExtensionApp):
+    """Disables and uninstalls all parts of this extension"""
+    name = "jupyter declarativewidgets quick-remove"
+    version = __version__
+    description = "Disables and removes all features of the jupyter_declarativewidgets extension"
+
+    def start(self):
+        self.argv.extend(['--py', 'declarativewidgets'])
+
+        from notebook import nbextensions
+        enable = nbextensions.DisableNBExtensionApp()
+        enable.initialize(self.argv)
+        enable.start()
+        install = nbextensions.UninstallNBExtensionApp()
+        install.initialize(self.argv)
+        install.start()
+        from notebook import serverextensions
+        install = serverextensions.DisableServerExtensionApp()
+        install.initialize(self.argv)
+        install.start()
+
 class ExtensionApp(Application):
     '''CLI for extension management.'''
     name = u'jupyter_declarativewidgets extension'
     description = u'Utilities for managing the jupyter_declarativewidgets extension'
     examples = ""
 
-    subcommands = dict(
-        install=(
-            ExtensionInstallApp,
-            "Install the extension."
-        ),
-        activate=(
-            ExtensionActivateApp,
-            "Activate the extension."
-        ),
-        deactivate=(
-            ExtensionDeactivateApp,
-            "Deactivate the extension."
-        ),
-        installr=(
-            ExtensionInstallRApp,
-            "Install the R extension"
-        )
-    )
+    subcommands = dict(installr=(
+        ExtensionInstallRApp,
+        "Install the R extension"
+    ))
+
+    if _new_extensions:
```

Comment:
That is weird. Then what is not right in my `dev_image_4.2` image. Cause when I the same in that image I get

```
jovyan@a2873ef7cec0:~/work$ jupyter declarativewidgets --help
Utilities for managing the jupyter_declarativewidgets extension

Subcommands
-----------

Subcommands are launched as `jupyter_declarativewidgets extension cmd [args]`.
For information on using subcommand 'cmd', do: `jupyter_declarativewidgets
extension cmd -h`.

quick-remove
    Disable and uninstall everything in the package
installr
    Install the R extension
quick-setup
    Install and enable everything in the package

Options
-------

Arguments that take values are actually convenience aliases to full
Configurables, whose aliases are listed on the help line. For more information
on full configurables, see '--help-all'.

--log-level=<Enum> (Application.log_level)
    Default: 30
    Choices: (0, 10, 20, 30, 40, 50, 'DEBUG', 'INFO', 'WARN', 'ERROR', 'CRITICAL')
    Set the log level by value or name.

To see all available configurables, use `--help-all`
``